common/src/test/java/org/apache/sedona/common/raster/RasterEditorsTest.java

```diff
@@ -87,4 +89,183 @@ public void testSetGeoReferenceWithEmptyRasterSRID() throws FactoryException {
         expected = "10.000000 \n3.000000 \n3.000000 \n-10.000000 \n15.000000 \n-7.000000";
         assertEquals(expected, actual);
     }
+
+
+    @Test
+    public void testResample() throws FactoryException, TransformException {
+        double[] values = {1, 2, 3, 4, 5, 6, 7, 8, 9};
+        GridCoverage2D raster = RasterConstructors.makeEmptyRaster(1, "d", 3, 3, 0, 0, 2, -2, 0, 0, 0);
```

Comment:
For the test cases, please always create EmptyRaster with non-equal width and height

Reply:
Done